### ISSUE TITLE
TKLBAM ideally should log restore rollback

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#481]
As per subject. Mentioned on the [forums]().

**Comment by lirazsiri** [comment]
Agreed. Shouldn't be a problem to implement that. I always assumed rollback would be interactive but maybe not the safest assumption to make.
